common/server-sent-events.conf

```diff
@@ -4,5 +4,5 @@ proxy_cache off;
 # make event source work with nginx
 # http://stackoverflow.com/questions/13672743/eventsource-server-sent-events-through-nginx
 proxy_set_header Connection '';
-proxy_http_version 1.1;
 chunked_transfer_encoding off;
+proxy_http_version 1.1;
```

Comment:
Please re-add it into the same place where it was removed from. Otherwise, this obfuscates the diff output. Thanks!